src/ReadableTimeInterval/ReadableTimeIntervalGenerator.php

```diff
@@ -61,6 +62,10 @@ public function getReadableTimeText(int $secondsToTransform)
             $textComponents[] = $this->translator->transChoice('timeInterval.seconds', $seconds, ['%count%' => $seconds]);
         }
 
+        if( true === empty($textComponents) ){
```

Comment:
isn't this the last check from 61?

Reply:
Returned value was '{space}and{space}' when zero was received